backend/src/controllers/user.ts

```diff
@@ -1,29 +1,19 @@
 import { Request, Response, NextFunction } from "express";
-import User from "../models/user.model";
+import {
+  handleSignup,
+  handleLogin,
+  handleUpdatePassword,
+  handlerRmoveUser,
+  handleLogout,
+} from "../handlers/user";
 
 export const signup = async (
   req: Request,
   res: Response,
   next: NextFunction
 ) => {
   try {
-    const user = await User.findOne({ where: { email: req.body.email } });
-    if (user) {
-      res.status(403).send({
-        message: "User " + req.body.email + " already exists!",
-      });
-    } else {
-      const newUser = await User.create({
-        email: req.body.email,
-        fullname: req.body.fullname,
-        password: req.body.password,
-      });
-
-      if (newUser) {
-        res.status(200);
-        res.json({ status: "Resgistration Successful!" });
-      }
-    }
+    handleSignup(req, res);
```

Comment:
Like we said in our conversation this morning, to the handler you don't pass the `req` and `res` objects, but only the parameters which the handler method needs.